topdown/aggregates.go

```diff
@@ -206,6 +206,46 @@ func builtinAny(a ast.Value) (ast.Value, error) {
 	}
 }
 
+func builtinMember(_ BuiltinContext, args []*ast.Term, iter func(*ast.Term) error) error {
+	containee := args[0]
+	switch c := args[1].Value.(type) {
+	case ast.Set:
+		return iter(ast.BooleanTerm(c.Contains(containee)))
+	case *ast.Array:
+		ret := false
+		c.Until(func(v *ast.Term) bool {
+			if v.Value.Compare(containee.Value) == 0 {
+				ret = true
+			}
+			return ret
+		})
+		return iter(ast.BooleanTerm(ret))
+	case ast.Object:
+		ret := false
+		c.Until(func(_, v *ast.Term) bool {
+			if v.Value.Compare(containee.Value) == 0 {
+				ret = true
+			}
+			return ret
+		})
+		return iter(ast.BooleanTerm(ret))
+	}
+	return nil
```

Comment:
The builtins still fail (not return `false`) if they're called with non-collection arguments in the last position. I think that's alright, what do you think?

Reply:
I was just wondering what my intuition was regarding _"Is 1 an element of 123?"_ and while I am leaning towards undefined, wolfram says [it's false](https://www.wolframalpha.com/input/?i2d=true&i=1+element+123). Let's do _false_ then, I'll update.